client/src/AppBundle/Resources/translations/ndr-assets.en.yml

```diff
@@ -18,9 +18,9 @@ existPage:
       label: Save and continue
 
 typePage:
-  htmlTitle: "New deputy report - assets | GOV.UK"
-  supportTitle: Assets
+  htmlTitle: "Add an asset - Complete the deputy report | GOV.UK"
```

Comment:
I believe on this page you choose your type of asset. Is there a better name we can give this?

Reply:
I don't believe so. The page title reads 'Add an asset' which makes sense when choosing a type of asset to add to the form.